salt/states/powerdns.py

```diff
@@ -0,0 +1,122 @@
+"""
+Create, modify and delete PowerDNS zone
+"""
+
+import logging
+
+from salt.exceptions import SaltInvocationError
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    return "powerdns.manage_zone" in __salt__ and "powerdns.delete_zone" in __salt__
```

Comment:
This is not a review blocker, nor does it need to change, but if one function is present, then the module loaded and the second is too.

Reply:
Thanks, I'll keep that in mind during working on a next module.